website/src/pages/docs/index.js

```diff
@@ -1,39 +1,9 @@
 import React from 'react';
-import Markdown from 'markdown-to-jsx';
-import classnames from 'classnames';
-import useBaseUrl from '@docusaurus/useBaseUrl';
-import Layout from '@theme/Layout';
-import items from '@site/src/data/docs';
-import styles from './styles.module.css';
+import { Redirect } from '@docusaurus/router'
 
 function Docs() {
```

Comment:
I didn't find the old docs landing page very useful. I think it will be better just merged into the main docs page